packages/svelte/src/main/main-client.js

```diff
@@ -185,7 +185,7 @@ function init_update_callbacks() {
 					user_effect(() => {
 						called_before = false;
 						called_after = true;
-						untrack(() => update_callbacks.after.forEach(/** @param {any} c */ (c) => c()));
+						update_callbacks.after.forEach(/** @param {any} c */ (c) => c());
```

Comment:
I don't think this is the correct fix. This means that any signal reads inside `update_callbacks` are registered with our `user_effect` so it reruns whenever those change.

Reply:
Ah got it. I will re-fix it.

Reply:
I partly take that back - in order to fix all possible cases, we need to track the signals inside before/afterUpdate, but in a manner that they rerun at the correct time and not too often

Reply:
@dummdidumm Can you give me an example of an unnecessarily frequent rerun the before/afterUpdate?